src/ImageRunModal.jsx

```diff
@@ -824,222 +824,224 @@ export class ImageRunModal extends React.Component {
+                                            <Button variant="plain" hasNoPadding aria-label="More info" icon={<OutlinedQuestionCircleIcon />} />
+                                        </Popover>
+                                    }>
+                                    <Flex alignItems={{ default: 'alignItemsCenter' }} className="ct-input-group-spacer-sm modal-run-limiter" id="run-image-dialog-cpu-priority">
+                                        <Checkbox id="run-image-dialog-cpu-priority-checkbox"
+                                            isChecked={this.state.cpuSharesConfigure}
+                                            onChange={(_event, checked) => this.onValueChanged('cpuSharesConfigure', checked)} />
+                                        <NumberInput
+                                            id="run-image-cpu-priority"
+                                            value={dialogValues.cpuShares}
+                                            onClick={() => !this.state.cpuSharesConfigure && this.onValueChanged('cpuSharesConfigure', true)}
+                                            min={2}
+                                            max={262144}
+                                            isDisabled={!this.state.cpuSharesConfigure}
+                                            onPlus={() => this.onPlusOne('cpuShares')}
+                                            onMinus={() => this.onMinusOne('cpuShares')}
+                                            minusBtnAriaLabel={_("Decrease CPU shares")}
+                                            plusBtnAriaLabel={_("Increase CPU shares")}
+                                            onChange={ev => this.onNumberValue('cpuShares', ev.target.value, 2)} />
+                                    </Flex>
                                 </FormGroup>
                             }
-                        </Grid>
-                        }
+                            {((userLingeringEnabled && userPodmanRestartAvailable) || (this.isSystem() && podmanRestartAvailable)) &&
+                                <Grid hasGutter md={6} sm={3}>
+                                    <GridItem>
+                                        <FormGroup fieldId='run-image-dialog-restart-policy' label={_("Restart policy")}
+                                            labelHelp={
+                                                <Popover aria-label={_("Restart policy help")}
+                                                    enableFlip
+                                                    bodyContent={userLingeringEnabled ? _("Restart policy to follow when containers exit. Using linger for auto-starting containers may not work in some circumstances, such as when ecryptfs, systemd-homed, NFS, or 2FA are used on a user account.") : _("Restart policy to follow when containers exit.")}>
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2246-57fb90ba-20250901-133037-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)